include/crill/platform.h

```diff
@@ -11,7 +11,7 @@
   #define CRILL_ARM 1
   #define CRILL_32BIT 1
   #define CRILL_ARM_32BIT 1
-#elif defined (__arm64__)
+#elif defined (__arm64__) || defined (_M_ARM64)
```

Comment:
While we’re at it would be great to add support for Android with `__aarch64__ ` as well (see [docs](https://developer.android.com/ndk/guides/cpu-features)).
```suggestion
#elif defined (__arm64__) || defined (__aarch64__) || defined (_M_ARM64)
```